Dockerfile

```diff
@@ -62,13 +67,10 @@ RUN apt-get update \
     && apt-get clean autoclean \
     && rm -fr /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
 
-# provides patch releases for 1.11 in /usr/local/bin/ninja
-RUN python3 -m pip install ninja~=${NINJA_MINOR_VERSION}
-
 RUN curl -sSLf https://apt.llvm.org/llvm-snapshot.gpg.key \
     | gpg --dearmor --output /usr/share/keyrings/llvm-snapshot.gpg \
     && chmod +r /usr/share/keyrings/llvm-snapshot.gpg \
-    && echo "deb [signed-by=/usr/share/keyrings/llvm-snapshot.gpg] http://apt.llvm.org/bionic/ llvm-toolchain-bionic-17 main" > /etc/apt/sources.list.d/llvm-snapshot.list
+    && echo "deb [signed-by=/usr/share/keyrings/llvm-snapshot.gpg] http://apt.llvm.org/focal/ llvm-toolchain-focal-17 main" > /etc/apt/sources.list.d/llvm-snapshot.list
 
 # when we migrate this builder to focal or newer, just remove this ppa and the rest should work
```

Comment:
can we do this?

Reply:
Yes, and the motivation is that a much newer llvm is available there than in the main Focal repo.